sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" />
```

Comment:
I waffled on where to host the snippets and almost crossed them into the Processor tests.  Ultimately, I decided to take the dependency here but without the full Storage test environment.   If we decide to write more comprehensive tests against the sample, we'll need to revisit.